src/c++/uds/userLinux/tests/albtest.c

```diff
@@ -289,6 +289,7 @@ TestResult runTest(const CU_SuiteInfo *suite, const CU_TestInfo *test)
     .tests = 1,
   };
 
+  bool previous = set_exit_on_assertion_failure(true);
```

Comment:
This works, but I think it would be more efficient to set it once near the start of the program rather than setting it for each test.Say, in main() right before figuring out what test modules to load.

(The flag is a global static, so it'll persist as long as the program is running, I think.)

Reply:
moved to main()